### PR TITLE
workflow

### PR DESCRIPTION
**Pull request by Varoothini28**
New tab - 'Understanding the ValQ workflow' added under Introduction

### CODE REVIEW

File: docs/intro/import.mdx [hunk 1]
```diff
@@ -12,99 +12,124 @@ You can use any of the three following ways to get ValQ added to Microsoft Power
...
 
+<div style={{textAlign: 'center'}}>
 <img alt="ValQ get" src={useBaseUrl('/doc-images/intro/install1.png')} 
     width={430}
-    height={395}
-/>
+    height={395}/>
+</div>
 
 ## 1. Import from file
```
Comment (vbisrikkanth) [inline docs/intro/import.mdx:21]: Can we move this section as the last section? 
1. App Source
2. My Organization
3. File

File: docs/intro/import.mdx [hunk 2]
```diff
@@ -12,99 +12,124 @@ You can use any of the three following ways to get ValQ added to Microsoft Power
...
-## 3. Import from AppSource - My Organization
- 
-The third and final way of importing ValQ is to get the app from **My Organization** tab found inside the Appsource.This method
-is applied when the admin has enabled ValQ as an app in your business.
-
-3. a) To get started, click on the three dots option **Import a custom visual** under the **Visualizations** pane
-   and choose **Import from AppSource** option as shown in the image below.
-
+## 3. Import from AppSource - My Organization 
+This option is used to download ValQ from the public AppSource marketplace. <br /> <br />
```
Comment (vbisrikkanth) [inline docs/intro/import.mdx:92]: This option is used to download ValQ if it's been as your organization visual.

**Comment by vbisrikkanth** [comment]
I have some questions on the wordings used like "Chapter". We can remove that because this might be from the user guide and it doesn't need to be the product documentation

**Comment by Varoothini28** [comment]
Hi Praveen

We shall use the word 'section' in the place of 'chapter' to be consistent and regarding the other suggestions, kindly commit the file for now, I will consider the changes with Jay and Chandra. Later will get back to you.

Thank you
Varoothini Shanmugasundaram

Functional Analyst, Products

[cid:70f6f465-acd7-43f6-a5ca-883745110586]<https://visualbi.com/>

www.visualbi.com<http://www.visualbi.com/>

[Twitter]<https://twitter.com/visual_BI>  [Facebook] <https://www.facebook.com/VISUALBI>   [LinkedIn] <https://www.linkedin.com/company/2406610>   [YouTube] <https://www.youtube.com/channel/UCo6ugHLh9kx9xAMetz-d8rA>



________________________________

**Comment by Varoothini28** [comment]
We shall use the word 'section' in the place of 'chapter' to be consistent and regarding the other suggestions, kindly commit the file for now, I will consider the changes with Jay and Chandra. Later will get back to you.

**Pull request merged**